_includes/body-footer.html

```diff
@@ -1,6 +1,7 @@
 <div class='wrapper--footer js-wrapper'>
-<div class='row row--sitemap' id='js-footer-nav'>
-<div class='row__inner row__inner--footer row--sitemap__inner'>
+<nav aria-label='Sitemap' class='row row--sitemap' id='js-footer-nav' itemscope='itemscope' itemtype='http://schema.org/SiteNavigationElement' role='navigation'>
```

Comment:
This content comes from Rizzo so you'd have to check that repo to see where this is being used.